### ISSUE TITLE
Clementine jams at the start of a song

### ISSUE THREAD

**Issue by Njadd** [clementine-player/Clementine#4978]
Sometimes, at the start of a song, neither the timer or the sound will start, and the task manager shows larger processor usage than usual. The song just never starts, I once left it on overnight and nothing changed. If I try to do anything (stop, skip, double-click another song), it will usually freeze.

It happened often under Windows Vista, I could sometimes solve the issue by clicking the Play button, but sometimes that would cause Clementine to freeze. Under Windows 7, it was all the time, and clicking Play would freeze 100% of the time. Under Windows 10, I'd say it's between the two. I never had a problem under Ubuntu.

Lowering the buffer time to 100-50ms seems to help, but not solve, and some songs will play in slow motion.

I don't know if anyone is reading this, but I do make a point of putting it out there because Clementine is the best music player I've ever had and it's really frustrating to see it hindered by such an issue.

**Comment by Bigbull3** [comment]
I have the same problem (HP notebook, Windows 7) so I installed the 1.1 version of Clementine instead of the 1.2, and now it works properly.

**Comment by Njadd** [comment]
Allright, thanks for that. Hope some dev sees this so a fix can be included in the next version

**Comment by Chocobozzz** [comment]
Did you try with the nightly builds ? http://builds.clementine-player.org/

**Comment by Njadd** [comment]
No, because I read that developper builds could damage user settings. I'd rather just deal with a few jams :)
(I mean, is it a developper build?)

**Comment by TheUbuntuGuy** [comment]
It is unlikely, but if you are worried, you can backup the settings by exporting the registry keys in ```HKEY_CURRENT_USER\Software\Clementine``` before you update.

**Comment by xiongchiamiov** [comment]
For what it's worth, I've been running nightlies (on OS X) for a few years now and haven't ever encountered a settings loss.  But YMMV.

**Comment by Redwan13** [comment]
I have the same problem with stable 1.2 version on os x 10.10.5.
